url/url.cpp

```diff
@@ -1460,14 +1464,16 @@ std::optional<std::tuple<std::uint64_t, bool>> UrlParser::parse_ipv4_number(std:
         }
     }
 
+    // TODO(zero-one): Differ width based on largest integer value supported by platform?
```

Comment:
Won't an u32 always be enough for an ipv4-address?

Reply:
yes, but some of the parser tests expect an individual number to be parsed correctly while failing the URL as a whole. i.e, this implementation has to deviate from the spec unless we use some kind of bignum library to allow numbers with an indefinite number of digits.